### DISCUSSION TITLE
Notify Remaining Steps and What Page Function

### DISCUSSION THREAD

**Discussion by penoptech** [Get-Your/Get-Your#96] [General]
@TiCampbell - I was testing some changes related to eligibility programs and started thinking about the way we currently handle redirecting users their in progress application. Below is a screen shot of our current flow and what I propose we should do. Ultimately, I'd like us to get rid of the 'Notify' page, but I can see an argument for keeping it around. It would probably be a jarring experience for a user to get dropped in the middle of an in progress application, but if we go with the flow I'm proposing it'd be fairly easy to incorporate a component that orientates them to the page they're on.

<img width="1294" alt="Screenshot 2023-03-26 at 9 47 50 AM" src="https://user-images.githubusercontent.com/55520784/227788806-37d0361a-f115-43e2-bdb5-49675d8d8c5f.png">
<img width="707" alt="Screenshot 2023-03-26 at 10 16 25 AM" src="https://user-images.githubusercontent.com/55520784/227789272-e188efed-20dd-4500-b6e8-ff236bc4070b.png">

**Comment by TiCampbell** [comment]
See #312 for followup.
